components/feature/prompts/src/main/java/mozilla/components/feature/prompts/PromptContainer.kt

```diff
@@ -51,4 +52,10 @@ internal sealed class PromptContainer {
 
         override fun getString(res: Int) = fragment.getString(res)
     }
+
+    @VisibleForTesting
+    internal class TestPromptContainer(override val context: Context) : PromptContainer() {
```

Comment:
We should be able to move this somewhere under `src/test`.

Reply:
Or is it because you don't want to open up `PromptContainer`?

Reply:
Yes

Reply:
OK, let's keep it then.